examples/example-node-connection-and-error/src/realm-query.ts

```diff
@@ -0,0 +1,119 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { BSON } from "realm";
+
+import { SYNC_STORE_ID } from "./atlas-app-services/config";
+import { Store } from "./models/Store";
+import { Kiosk } from "./models/Kiosk";
+import { Product } from "./models/Product";
+import { getRealm } from "./realm-auth";
+
+export function getStore() {
+  return getRealm()?.objects(Store).filtered("_id = $0", SYNC_STORE_ID)[0];
+  // or:            .objects<Store>(Store.schema.name, ..)
+}
+
+function getKiosks() {
+  const realm = getRealm();
+  return realm ? realm.objects<Kiosk>(Kiosk).filtered("storeId = $0", SYNC_STORE_ID) : [];
+}
+
+function getProducts() {
+  const realm = getRealm();
+  return realm ? realm.objects<Product>(Product).filtered("storeId = $0", SYNC_STORE_ID) : [];
+}
+
+function addProducts() {
+  const realm = getRealm();
+  if (realm) {
+    realm.write(() => {
+      const NUM_PRODUCTS = 10;
+      for (let i = 1; i <= NUM_PRODUCTS; i++) {
+        const randomPrice = parseFloat((5 + Math.random() * 10).toFixed(2));
+        realm.create(Product, { // Or: `realm.create<Product>(Product.schema.name, ..)`
+          _id: new BSON.ObjectId(),
+          storeId: SYNC_STORE_ID,
+          name: `product${i}`,
+          price: randomPrice,
+          numInStock: NUM_PRODUCTS,
+        });
+      }
+    });
+  }
+}
+
+function addKiosks() {
+  const realm = getRealm();
+  if (realm) {
+    const products = getProducts();
+    realm.write(() => {
+      for (let i = 1; i <= 10; i++) {
```

Comment:
Should `10` be a constant (like in `addProducts()`)?

Reply:
Good catch! Yes that'd be better, thanks 🙂